sequence_processing_pipeline/FastQCJob.py

```diff
@@ -92,6 +92,10 @@ def _find_projects(self, path_to_run_id_data_fastq_dir, is_raw_input):
             files = [x for x in files if x.endswith('.fastq.gz') and
                      'zero_files' not in x]
 
+            # remove fastq files in the only-adapter-filtered
+            # folder from consideration if they are present.
+            files = [x for x in files if 'only-adapter-filtered' not in x]
```

Comment:
my concern here is that this folder is at the top of the NuQCJob folder, which potentially means that if there are more than one project with the same filename it could be overwritten; is this possible?

Reply:
Correct, that's what this PR is fixing. As NuQCJob processes each project, it will move the adapter-trimmed files from only-adapter-filtered into a subdirectory of said folder. This way there will be no overwrites while NuQCJob is running.

This line here is in FastQCJob, which is downstream of NuQCJob. This line is the same line I added to qiita-rc. If the path to _find_projects is a sub-directory of NuQCJob, then this line will have no effect. If however it's passed the path to the entire working directory, this will safely cover that situation.